test/Closures/rlexe.xml

```diff
@@ -158,4 +158,10 @@
       <tags>BugFix</tags>
     </default>
   </test>
+  <test>
+    <default>
+      <files>bug_OS_9008744.js</files>
+      <tags>BugFix,exclude_dynapogo</tags>
```

Comment:
Fair enough. I wish we had a forcedeferparse/nodeferparse/forceundodefer nightly run :)